src/docs/base.mdx

```diff
@@ -105,11 +105,9 @@ links:
 
     | Opcode | Name | Solidity Equivalent | Rollup Behaviour | Ethereum L1 Behaviour |
     | :----- | :--- | :-------------------| :--------------- | :-------------------- |
-    | 49 | BLOBHASH | `blobhash(index)` | Undefined | Returns versioned hash of the `index`-th blob associated with the current transaction. <Unsupported /> |
```

Comment:
So do they return the latest imported L1 blobhash and blobbasefee?

Reply:
They return the blobbasefee of the L2 and the hash of the blob assocticated with the transaction